src/services/auth/auth.tsx

```
@@ -220,7 +220,7 @@ export const getDocumentsList = async () => {
 		);
 
 		// Return the documents list data
-		return response.data;
+		return response.data;	
```

ESLint

[error] Delete `↹`

(prettier/prettier)